parsons/ngpvan/van_connector.py

```diff
@@ -73,7 +73,7 @@ def get_request(self, endpoint, **kwargs):
         data = self.api.data_parse(r)
 
         # Paginate
-        while isinstance(r, dict) and self.api.next_page_check_url(r):
+        while isinstance(r, dict) and len(r['items']) > 0:
```

Comment:
I believe this is a temporary error on VAN's side. Unfortunately, I think your suggestion would interfere with all sorts of other methods relying on `next_page_check_url()`, and I can confirm that it's causing some unit tests to fail.

May I recommend switching it to the following hot fix, that is more easily ignored/removable once we get VAN to fix their bug?
```
while isinstance(r, dict) and self.api.next_page_check_url(r):
    if endpoint == 'savedLists' and not r['items']:
        break
```

Reply:
that works, thank you!! and appreciate you taking the time to look into this more carefully than I did